app/src/ui/diff/text-diff.tsx

```diff
@@ -1019,6 +1022,48 @@ export class TextDiff extends React.Component<ITextDiffProps, ITextDiffState> {
       : canSelect(file) && file.selection.isSelected(index)
   }
 
+  /**
+   * Returns a string with the pixel size for the gutter width.
+   * 1 = 20
+   * >1 = (10 * digit amount) + 5
+   */
+  private getGutterLineWidth(totalHunks: readonly DiffHunk[]): string {
+    // Get the largest old line number and the largest new line, of these two we can find the highest amount of digits in the hunks
+    const largestOldLineNum = Math.max(
+      ...totalHunks.map(function (o) {
+        return Math.max(
+          ...o.lines.map(function (val) {
+            return val.oldLineNumber ?? 0
+          })
+        )
+      })
+    )
+
+    const largestNewLineNum = Math.max(
+      ...totalHunks.map(function (o) {
+        return Math.max(
+          ...o.lines.map(function (val) {
+            return val.newLineNumber ?? 0
+          })
+        )
+      })
+    )
+
+    const maxLinesDigitAmount =
```

Comment:
Seems like duplicate logic for calculating width at the end of this method and the end of `getRowWidth` for side by side implementation, suggest it could be moved to a util/lib function.

Reply:
I moved this to diff-helper.tsx, but I'm not sure if this is the correct location for it. Let me know if I should move it elsewhere.